### DISCUSSION TITLE
Help to restore a link to a site for a customer on Github

### DISCUSSION THREAD

**Discussion by IrenArt** [community/community#31725] [Other Feature Feedback, Questions, & Ideas]
Hi everyone! I accidentally deleted the link to the site on the github for customer. The repository isn`t deleted. Now the customer sees a 404 error on the link. How to restore the link?

**Comment by Simran-B** [comment]
Did you delete a link or a file? You can use Git to revert the deletion of a file, which should restore the link to it.

**Comment by Kirtishukla2004** [comment]
hello 
Go to github contact page here and email them saying you need to recover your deleted gist and also mention the link or the description of the deleted gist.They will recover it for you. 
hope this will help
